configure.ac

```diff
@@ -1331,6 +1319,10 @@ if test "x${enable_systemd}" != xno; then
             enable_systemd=no
         fi
     fi
+    if test $(echo "$CPPFLAGS" | grep -q PCMK_TIME_EMERGENCY_CGT) \
+            || test "x$ac_cv_have_decl_CLOCK_MONOTONIC" = xno; then
+        AC_MSG_FAILURE([cannot enable systemd without clock_gettime(CLOCK_MONOTONIC, ...)])
```

Comment:
Quadrigraphs are needed to put parentheses in text strings:

    ( = @{:@
    ) = @:}@